tests/runtime_tests/string_utility.cpp

```diff
@@ -315,15 +314,15 @@ TEST_CASE("append ints", "[utility]") {
         // Different expectation at runtime and compile-time. At runtime,
         // we are stuck with snprintf, which insists on writing a null-terminator character,
         // therefore we loose one character at the end.
```

Comment:
nit: should we fix this comment now (and similar comments below)?

Reply:
Indeed, these comments can be removed now